guides/plugins/plugins/content/cms/real-world-cms/add-real-world-cms-block.md

```diff
@@ -0,0 +1,268 @@
+# Add a real world CMS block
+
+## Overview
+
+This guide will teach you how to create a CMS block with your plugin. This guide will cover a real world example to keep everything easy we will just create a  button. The CMS block for the button we will cover right here.
+
+## Prerequisites
+
+This plugin is built based on the [Add CMS block](../add-cms-block.md) guide. We assume that you know the basics for a CMS block from this guide.
```

Comment:
I like the style of this guide a lot. I think this guide can replace the old one if we copy-paste the parts that aren't covered :thinking:  @NinjaArmy @JoshuaBehrens Do you think this would make sense for end-users? I can also see the doubling of the information helping it stick.

Reply:
The information from the old guides is needed if it should replaces the old guides. I think having both guides helps the most.

Reply:
I re-read both guides. They overlap not completely as @NinjaArmy says. So it can't be just replacing. This would need more work to fuse them. But I see that it might be weird to have two similar CMS guides.

Reply:
The current guides are not coherent (element & block) and these guides are based on each other (element & block). So it probably helps to understand how blocks and elements work together.

So maybe it could be something like "How blocks & elements work together"? But we would still have 2 guides..